documentation/decisions/adr/2023-10-16-email-marketing.md

```diff
@@ -0,0 +1,215 @@
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                         | Sendy | Salesforce | MailChimp | Hubspot | Mautic | Adobe |
+| ---------------------------    | :---: | :--------: | :-------: | :-----: | :----: | :---: |
+| Usability                      |  2    |   1        |      3    |    3    |    1   |    3  |
+| Content Review                 |  🔄   |   ✅        |     ✅    |   ✅    |   ✅    |  ✅   |
+| Comments                       |   ❌  |   ✅        |     ❌    |   ✅    |   ✅    |   ✅  |
+| Open source*                   |  🔄   |    ❌       |     ❌    |   ❌    |   ✅    |   ❌  |
+| Multi-Media                    |   ✅  |   ✅        |     ❌    |   ✅    |   ✅    |   🔄  |
+| Analytics                      |    1  |   ✅        |      ✅   |    ✅   |   ✅    |   ✅  |
+| Internationalization           |     ❓ |   2        |      3    |    2   |    1   |   1   |
+| Onboarding costs               |     1 |   3        |      1    |    2    |    1    |    2 |
+| Community and Support*         |     1 |   3        |     3     |   3     |   3     |   3  |
+| Scalability                    |     🔄 |   ✅       |     ✅    |   ✅    |   ✅     | ❌    |
+| Send 1M emails quickly         |    ❌  |   ❓       |     ❓    |   ❓    |   ❓     |   ❓  |
+| Authority to Operate or FedRAMP|     ✅ |   ❓       |     ❌    |   ❌    |   ❌     |   ✅  |
+
+
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### Sendy
+
+#### Details
+
+Sendy is a self-hosted email marketing application that allows users to send newsletters, manage subscribers, and track campaign performance. It is known for its cost-effectiveness and integration with Amazon SES for efficient email delivery.
+
+#### Pros
+
+- Sendy is currently used by HHS and it would be simple for existing users to start using the tool right away
+- It provides decent metrics and analytics
+- HHS is able to send a few campaigns simultaneously
+- Sendy is self-hosted which means we have control over the email list and the data. Sendy is a self hosted application that runs on our own web server. Pay once and it's yours, there's no recurring fee.
+- It is a cost-effective option as it is self-hosted.
+- Self-hosting provides control over data and infrastructure
+
+#### Cons
+
+- Campaigns with larged audiences or number of emails take a long time to send and a user cannot use the tool while an email is being sent. Currently, we have 1M subscribers and it takes a long time to send to that many subscribers
```

Comment:
Could this improve with better infrastructure for the self-hosting? I'm not sure how MH is backing the tool right now.

Reply:
Yeah, they mentioned they took a couple actions to improve this. I added a sentence to include that.